src/method.rs

```diff
@@ -92,3 +92,19 @@ impl<'a> std::convert::TryFrom<&'a str> for Method {
         Self::from_str(value)
     }
 }
+
+impl AsRef<str> for Method {
```

Comment:
Is this required to wasm32-unknown-unknown?

Reply:
Strictly, no. But I needed it when fixing both http-client(https://github.com/jbr/http-client/pull/1) and surf(I'll open a pr in a few minutes) compilation.

Reply:
Surf pr: https://github.com/jbr/surf/pull/1

Reply:
If adding this implementation is a concern, I can remove it and do something else downstream. This just made the most sense IMO for how method was being used downstream.

Reply:
No objections to this bound.